test/helpers/registry/blobs.go

```diff
@@ -347,12 +350,12 @@ func (b *blobs) handle(resp http.ResponseWriter, req *http.Request) *regError {
 			}
 
 			err = bmh.Mount(req.Context(), repo, from, h)
-			if err != nil {
-				return regErrInternal(err)
+			if err == nil {
+				resp.Header().Set("Docker-Content-Digest", h.String())
+				resp.WriteHeader(http.StatusCreated)
+				return nil
 			}
-			resp.Header().Set("Docker-Content-Digest", h.String())
-			resp.WriteHeader(http.StatusCreated)
-			return nil
+			// check err type??
```

Comment:
a TODO? Or just thought? Looks like we are ignoring any error that the Mount() comes back with now.

Reply:
This is just a temporary pkg that we are waiting it to be merged in ggcr so we can remove it from imgpkg